scripts/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+TAG=$( cd src/github.com/SUSE/cf-plugin-backup
+       git describe --tags --long | awk -F - '{ print $1 }' )
+
+NAME=$( cd src/github.com/SUSE/cf-plugin-backup
+        basename $(git config --get remote.origin.url) .git)
+
+# Assemble configuration data for `out` to git-release resource.
+echo > release/name ${NAME}
+echo > release/tag  ${TAG}
+echo > release/body "${NAME}: Release ${TAG}"
+
+# Build the tarballs for the release
+make -C src/github.com/SUSE/cf-plugin-backup build
+make -C src/github.com/SUSE/cf-plugin-backup dist
```

Comment:
Having these here instead of in the normal step seems _really_ odd.

Reply:
Note that `release` is driven from the __tag-filtered__ stream of commits.
The normal step is not filtered.
`src-tags` vs `src`.

Reply:
Correct, hence me thinking we should duplicate pipelines (to have one that's tag-specific), instead of just putting it on the existing master one.